src/main/java/io/vertx/core/impl/launcher/commands/BareCommand.java

```diff
@@ -338,27 +348,6 @@ protected void beforeStartingVertx(VertxOptions options) {
     }
   }
 
-  /**
-   * @return the event bus options.
-   */
-  protected EventBusOptions getEventBusOptions() {
```

Comment:
If the PR is merged, this is a breaking change, it will have to be documented on the wiki

Reply:
can we do without a breaking change ?

Reply:
Ok, so I will keep this function